From f1bad83a21f0e1c8fe92f19f8eaa1b1ffd1bed9d Mon Sep 17 00:00:00 2001
From: Sam Lantinga <slouken@libsdl.org>
Date: Sat, 3 Jun 2023 11:36:38 -0700
Subject: [PATCH 414/458] Revert "Fixed detection of the Steam Virtual Gamepad
 on macOS"

This reverts commit 7aa28ce279045884653c8153a5cee99f4622af10.

Steam has been updated to send a version of 1 to avoid conflicts with controllers that report a version of 0.
---
 src/joystick/hidapi/SDL_hidapi_xbox360.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/joystick/hidapi/SDL_hidapi_xbox360.c b/src/joystick/hidapi/SDL_hidapi_xbox360.c
index d3f672296..63e8f9191 100644
--- a/src/joystick/hidapi/SDL_hidapi_xbox360.c
+++ b/src/joystick/hidapi/SDL_hidapi_xbox360.c
@@ -85,8 +85,7 @@ static SDL_bool HIDAPI_DriverXbox360_IsSupportedDevice(SDL_HIDAPI_Device *device
         return SDL_FALSE;
     }
 #if defined(__MACOSX__) || defined(__WIN32__)
-    if (vendor_id == USB_VENDOR_MICROSOFT && product_id == 0x028e &&
-        (version == 0 || version == 1)) {
+    if (vendor_id == USB_VENDOR_MICROSOFT && product_id == 0x028e && version == 1) {
         /* This is the Steam Virtual Gamepad, which isn't supported by this driver */
         return SDL_FALSE;
     }
-- 
2.17.1

